lte/cloud/go/services/lte/obsidian/handlers/handlers.go

```diff
@@ -812,6 +813,37 @@ func getGatewayPoolHandler(c echo.Context) error {
 	return c.JSON(http.StatusOK, gatewayPool)
 }
 
+func updateGatewayPoolHandler(c echo.Context) error {
+	networkID, gatewayPoolID, nerr := getNetworkIDAndGatewayPoolID(c)
+	if nerr != nil {
+		return nerr
+	}
+	gatewayPool := new(lte_models.MutableCellularGatewayPool)
```

Comment:
Nit: literals are the idiomatic way -- `... := &lte_models.MutableCellularGatewayPool{}`